pkg/apis/config/image_policies.go

```diff
@@ -41,13 +41,13 @@ type ImagePolicyConfig struct {
 	// in the Secret. So it seemed unnecessary to mirror those all over.
 	// Key in the map is the name of the ClusterImagePolicy where the Policy
 	// was received from.
-	Policies map[string]v1alpha1.ClusterImagePolicySpec
+	Policies map[string]internalcip.ClusterImagePolicySpec
```

Comment:
You can now probably remove my TODO since you did it :) Thanks!

Reply:
Awesome! removed